package.json

```diff
@@ -14,22 +14,16 @@
     "always-auth": true
   },
   "engines": {
-    "node": ">=8"
+    "node": "^14.15.4"
   },
```

Comment:
Careful, this is a library, so by specifying `^14.15.4`, we won't be able to upgrade to this version in any apps/libraries that use anything less than 14.15.4 (or someday 16, etc.).

This was here because we very specifically dropped support for node 6 many years ago. So we could either remove it (since it isn't worth specifying anymore, with little or none of our repos using node <=6) or we could leave it as is? Your call.